README.md

```diff
@@ -1,2 +1,29 @@
 # migration
 Migration tools for TiKV, e.g. online bulk load.
+
+# How to build from source
+
+```
+mvn clean package
+```
+# Quick Start
+```
+spark-submit tikv-client-java-3.2.0-SNAPSHOT.jar --class org.tikv.bulkload.example.BulkLoadExample migration-0.0.1-SNAPSHOT.jar <pdaddr>
+```
+
+# Configuration
```

Comment:
```suggestion
## Configuration
```

Reply:
Fixed in commit [[BulkLoad: fix README.md and keys must be added in order exception](https://github.com/tikv/migration/pull/1/commits/3c0520a1d4aefee81148474a77b47ff0fcacc6da)]